config/sql/migration/migrate.15.sql

```diff
@@ -0,0 +1,9 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+BEGIN TRANSACTION;
+
+CREATE TABLE delegations(meta BLOB NOT NULL, role_name TEXT NOT NULL, UNIQUE(role_name));
```

Comment:
Why not a new type in `meta` table?

Reply:
I went back and forth on this, but the main problem is that the `meta` table expects a `meta_type` integer, which we don't have for delegations. They can come and go over the lifespan of a device, so assigning them arbitrary numbers seems like unnecessary work. We also don't track versions on non-root metadata in the `meta` table as it is, so it seemed like more waste in the delegation case.

Furthermore, the `meta` table is fairly rigidly structured. We only replace items in it. It seemed like delegations were different enough to warrant a separate table with less stringent rules about the contents.